kfp/transform_workflows/.make.transforms_workflows

```diff
@@ -1,7 +1,50 @@
 include ${REPOROOT}/.make.versions
+include ${REPOROOT}/kfp/requirements.env
+
+# Include the common rules.
+# Use "make help" to see them.
+include ${REPOROOT}/.make.defaults
 
 DOCKER_REPO ?= ""
 
-.reconcile-requirement::
+define set_env_var
+	$(eval export $(1)=$(2))
+endef
+
+.PHONY: .transforms_workflows.reconcile-requirement
+.transforms_workflows.reconcile-requirement:
 	sed -i.back "s%${DOCKER_IMAGE_NAME}:.*%${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_VERSION}\"%" ${PIPELINE_FILE}
-	sed -i.back "s%kfp-data-processing:.*%kfp-data-processing:${KFP_DOCKER_VERSION}\"%" ${PIPELINE_FILE}
+	sed -i.back "s/kfp-data-processing:.*/kfp-data-processing:${KFP_DOCKER_VERSION}\"/" ${PIPELINE_FILE}
```

Comment:
Why do you use `/` as a sed command separator here, but' %' is used in the previous line?

Reply:
I will change it to % in all places. Thanks